task4/script.js

```diff
@@ -0,0 +1,702 @@
+        createdAt: new Date('2018-02-23T20:00:00'),
+        author: 'SamiyKrytoi',
+        photoLink: './img/7.jpg',
+        likes: [
+            'Ivan Ivanov',
+            'SamiyKrytoi',
+            'Egor1995',
+            'Ekaterina Sosinovich'
+        ],
+        hashTags: [
+            '#life',
+            '#hashtag2'
+        ]
+    },
+    {
+        id: '16',
+        description: 'Thanks for being such a nice team lead! Happy Birthday, Ivan!',
+        createdAt: new Date('2018-02-23T20:00:00'),
+        author: 'Egor1995',
+        photoLink: './img/15.jpg',
+        likes: [
+            'Ivan Ivanov',
+            'EgorGitar',
+            'Kate Sosinovich',
+            'Kate145K'
+        ],
+        hashTags: []
+    },
+    {
+        id: '1',
+        description: 'This is an amazing photo! Do you like it?',
+        createdAt: new Date('2018-02-23T20:00:00'),
+        author: 'Kate145',
+        photoLink: './img/0.jpg',
+        likes: [
+            'Ivan Ivanov',
+            'AnyaKotik',
+            'SamiyKrytoi',
+            'Egor1995'
+        ],
+        hashTags: [
+            '#life',
+            '#hashtag2'
+        ]
+    },
+    {
+        id: '2',
+        description: 'It\'s my cat Cookie. I love him sooo much',
+        createdAt: new Date('2018-02-24T14:34:22'),
+        author: 'AnyaKotik',
+        photoLink: './img/1.jpg',
+        likes: [
+            'Ivan Ivanov',
+            'Kate145',
+            'SamiyKrytoi',
+            'Egor1995',
+            'Natya Leonova',
+            'KirrilK'
+        ],
+        hashTags: [
+            '#my_cat',
+            '#fluffy'
+        ]
+    },
+    {
+        id: '6',
+        description: 'Can\'t stop gazing at this photo! I\'m dreaming of visiting such kind of place',
+        createdAt: new Date('2018-02-25T19:12:04'),
+        author: 'Kate Sosinovich',
+        photoLink: './img/5.jpg',
+        likes: [
+            'Ivan Ivanov',
+            'AnnaKirova',
+            'SamiyKrytoi',
+            'EgorGitar'
+        ],
+        hashTags: [
+            '#landscape',
+            '#nature',
+            '#life',
+            '#trip',
+            '#dream'
+        ]
+    },
+    {
+        id: '19',
+        description: 'Tulips and roses. Missing spring so much!!!',
+        createdAt: new Date('2018-02-28T17:45:02'),
+        author: 'Kate145',
+        photoLink: './img/18.jpg',
+        likes: [
+            'AnyaKotik',
+            'Svetik1998',
+            'Kate Sosinovich'
+        ],
+        hashTags: [
+            '#spring',
+            '#flowers',
+            '#roses',
+            '#tulips'
+        ]
+    }];
+
+//- Module
+let photoActions = (function () {
+    let getPhotoPosts = (skip, top, filterConfig = {}) => {
+        skip = skip ? skip : 0;
+        top = top ? top : 10;
+        let resPhotoPosts = photoPosts;
+        if (filterConfig.length !== 0) {
```

Comment:
length?

Reply:
fixed. or it's better to remove this check at all?